src/polycap.c

```diff
@@ -1,7 +1,9 @@
 #include "config.h"
 #ifdef _WIN32
+  #ifndef _CRT_RAND_S
```

Comment:
mss niet helemaal zuiver/veilig om te doen, maar kwas bang dat we anders compilatie problemen gingen krijgen eens we polycap.c niet meer nodig hebben. Mijn grootste twijfel is hoe we forceren dat _crt_rand_s zeker gedefinieerd wordt voor stdlib.h wordt geïncludeerd. Nu geen probleem daar polycap.c vooraan in de makelist staat, maar moeten we later goed op letten.

Reply:
Dat is eigenlijk niet nodig. Als het al gedefinieerd is zal opnieuw definiëren geen gevolgen hebben.

De volgorde in de Makefile doet er niet toe: alle .c files worden allemaal apart gecompileerd, onafhankelijk van elkaar. Eigenlijk is er zelfs geen garantie over de volgorde zodra je `make` oproept met de `-j` optie.

Als je op termijn de code met de `rand_s` functie verhuist naar een ander bestand, zal je er enkel voor moeten zorgen dat de `<stdlib.h>` als eerste header wordt ge-`#include`-d, net na definiëren van `_CRT_RAND_S`. Ik heb al gemerkt dat vele andere headers zelf ook `<stdlib.h>` gaan includen...